app/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "app",
+  "version": "1.0.0",
+  "description": "Netflix roulette",
+  "main": "index.js",
+  "scripts": {
+    "start": "start",
```

Comment:
For me "npm start" command opens a new terminal window :) I expected this to run app using webpack dev server